src/components/NavBar.jsx

```diff
@@ -20,7 +20,7 @@ export function NavBar({ rulesSatisfied }) {
   return (
     <Navbar bg="light" className="control-panel" expand="lg">
       <Container fluid className="ps-0 pe-0">
-        <Nav.Link as={NavLink} to="" end>
```

Comment:
Should we consider using absolute paths in all places - e.g. [here](https://github.com/janzarsky/scout-planner/blob/fe42a4a12131fb155e5d4ed517f02fa85ee8d955/src/components/TimetableV2.tsx#L950)?

Reply:
probably.. I was not able to figure out a way to do it using relative paths (since navbar is used both on `/:table` and `/:table/rules` level. But currently it works